schemas/atlas/brainAtlasVersion.schema.tpl.json

```diff
@@ -26,10 +26,10 @@
         "legalPerson"
       ]
     },  
-    "coordinateSpace": {
-      "_instruction": "Add the common coordinate space in which this brain atlas version exists in.",
+    "coordinateSpaceVersion": {
+      "_instruction": "Add the common coordinate space version in which this brain atlas version exists in.",
       "_linkedTypes": [
-        "https://openminds.ebrains.eu/sands/CommonCoordinateSpace"
+        "https://openminds.ebrains.eu/sands/CommonCoordinateSpaceVersion"
       ]
     },
```

Comment:
I would probably leave it at coordinateSpace. This would be the only exception and just "coordinateSpace" would still not be wrong to use even though we would only expect CCS versions. Noticed this while working on the draft for the annotation updates where atlasAnnotation could have gotten "coordinateSpaceVerison" as property name, but then it would be inconsistent across annotations. CustomAnnotation cannot get the same one because of the "CustomCoordinateSpace" that is not versioned.

Reply:
done

Reply:
Yeah, I say that same "issue" while preparing for the repo-wide clean-up. I'll fix it along with those changes.